apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -59,12 +61,20 @@ const Auth = (): JSX.Element => {
 		return null;
 	};
 
+	const handleImageError = useCallback(() => {
+		setIsImageLoaded(false);
+	}, []);
+
 	return (
 		<main className={styles["container"]}>
 			<section className={styles["auth-container"]}>
 				<div className={styles["left-side"]}>
-					{/* TODO: replace logo */}
-					<img alt="logo" className={styles["logo-wrapper"]} src="" />
+					{/* TODO: add logoSrc */}
+					<Logo
+						isImageLoaded={isImageLoaded}
+						logoSrc=""
+						onImageError={handleImageError}
+					/>
```

Comment:
Too complicated logic for this case 😅 let's keep it as it was, the only thing that is needed is to use a random image as a placeholder. You can download "LOGO" from the design as an image for this.

Reply:
I think we don't need to handle case when image is not loaded, the application won't build if we we will import invalid image path

Reply:
It was in case we didn't import path at all :sweat_smile:

Reply:
Oh.. I was totally thinking the same thing and asked about it earlier, but unfortunately, you didn't see it then. Okay, I'll just revert it to how it was :black_cat:

Reply:
oh sorry, probably missed that question

Reply:
no worries :smile_cat: :